PeerioTechnologies/peerio-desktop
# feat: remember opened folder when switching to/from Files view

#### Relevant info and issue/PR links
https://app.clubhouse.io/peerio/story/14684/desktop-remember-folder-when-switching-back-to-files-tab

#### Testing instructions
Go to Files, open a folder, navigate away from Files and then back again, same folder should remain open.

----
### Repository owner


[PR guideline](https://github.com/PeerioTechnologies/peerio-icebear/blob/dev/docs/CONTRIBUTING.md)
